blocks/inspector-controls/select-control/index.js

```diff
@@ -14,23 +14,22 @@ import { withInstanceId } from 'components';
 import BaseControl from './../base-control';
 import './style.scss';
 
-function SelectControl( { label, selected, instanceId, onBlur, options = [], ...props } ) {
+function SelectControl( { label, selected, instanceId, onChange, options = [], ...props } ) {
```

Comment:
There's a lint warning that `selected` is unused.

I think we'll want to promote the idea of [Controlled Components](https://facebook.github.io/react/docs/forms.html#controlled-components) for inputs and encourage the developer to pass the value explicitly.

The React docs touch on the issue that existed here, and their workaround:

https://facebook.github.io/react/docs/forms.html#the-select-tag